<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit_map_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>메모장에 적은후 복사하세요.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Map</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_dfs">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;</string>
   </property>
   <property name="text">
    <string>DFS</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_bfs">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>BFS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calc">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>계산</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_x">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>입구 좌표</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_y">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_x">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_y">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>출구 좌표</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 200);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>벽</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 100, 100);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>방문
경로</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 255, 200);
color: rgb(255, 255, 255);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>답
경로</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 0, 0, 200);
color: rgb(255, 255, 255);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>답
경로</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(100, 0, 0, 100);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>방문
경로</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>20</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>32</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_dfs_step">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>DFS Step</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bfs_step">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>BFS Step</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 100, 0, 100);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>방문
경로</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_best_step">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>BestFirst Step</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_best">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:green;</string>
   </property>
   <property name="text">
    <string>BestFirst</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 255, 0, 200);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>답
경로</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_dfs_visit_step">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Visit Step</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bfs_visit_step">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Visit Step</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_best_visit_step">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Visit Step</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
